subprojects/packagefiles/qarchive/meson.build

```diff
@@ -0,0 +1,69 @@
+project(
+  'QArchive',
+  'cpp',
+  version: '2.1.1',
+  license: 'BSD-3-Clause',
+  default_options: ['cpp_std=c++11'],
+  meson_version: '>=0.59.0') # because of compile_moc
+
+qt5 = import('qt5')
+
+libarchive_dep = dependency('libarchive')
+qt5core_dep = dependency('qt5', modules: ['Core'])
+
+configure_file(
+  input: 'other/meson/config.h.in',
+  output: 'config.h',
+  configuration: configuration_data({'QARCHIVE_STATIC': get_option('static')}))
+
+src = files(
+  'src/qarchive_enums.cc',
+  'src/qarchiveutils_p.cc',
+  'src/qarchivememoryfile.cc',
+  'src/qarchivememoryextractoroutput.cc',
+  'src/qarchiveioreader_p.cc',
+  'src/qarchiveextractor_p.cc',
+  'src/qarchiveextractor.cc',
+  'src/qarchivecompressor_p.cc',
+  'src/qarchivecompressor.cc',
+  'src/qarchivediskextractor.cc',
+  'src/qarchivememoryextractor.cc',
+  'src/qarchivediskcompressor.cc',
+  'src/qarchivememorycompressor.cc')
+
+headers = files(
+  'include/qarchive_enums.hpp',
+  'include/qarchiveutils_p.hpp',
+  'include/qarchivememoryfile.hpp',
+  'include/qarchivememoryextractoroutput.hpp',
+  'include/qarchiveioreader_p.hpp',
+  'include/qarchiveextractor_p.hpp',
+  'include/qarchiveextractor.hpp',
+  'include/qarchivecompressor_p.hpp',
+  'include/qarchivecompressor.hpp',
+  'include/qarchivediskextractor.hpp',
+  'include/qarchivememoryextractor.hpp',
+  'include/qarchivediskcompressor.hpp',
+  'include/qarchivememorycompressor.hpp',
+  'include/qarchive_global.hpp')
+
+include_dirs = ['include', '.']
```

Comment:
I’d use include_directories instead of an array.

Reply:
It makes no difference unless you access the variable from another directory's `meson.build` file.

Reply:
Sure. But it’s good practice